thorlabs_apt_protocol/functions.py

```diff
@@ -2183,3 +2183,198 @@ def pol_set_params(
+    return _pack(0x0261, dest, source, param1=chan_ident)
+
+
+def mot_set_ioconfig(
+    dest: int, source: int, io_port: int, mode: int, out_source: int
+) -> bytes:
+    data = struct.pack("<HHH", io_port, mode, out_source)
+    return _pack(0x0263, dest, source, data=data)
+
+
+def mot_req_ioconfig(dest: int, source: int, chan_ident: int, io_port: int) -> bytes:
+    return _pack(0x0264, dest, source, param1=chan_ident, param2=io_port)
+
+
+def mot_set_auxioconfig(
+    dest: int, source: int, out_port: int, mode: int, sw_source: int
+) -> bytes:
+    data = struct.pack("<HHH", out_port, mode, sw_source)
+    return _pack(0x0266, dest, source, data=data)
+
+
+def mot_req_auxioconfig(
+    dest: int, source: int, chan_ident: int, out_port: int
+) -> bytes:
+    return _pack(0x0267, dest, source, param1=chan_ident, param2=out_port)
+
+
+def mot_set_analogmonitorconfig(
+    dest: int,
+    source: int,
+    monitor: int,
+    motor_channel: int,
+    sys_var: int,
+    scale: int,
+    offset: int,
+) -> bytes:
+    data = struct.pack("<HHHll", monitor, motor_channel, sys_var, scale, offset)
+    return _pack(0x0269, dest, source, data=data)
+
+
+def mot_req_analogmonitorconfig(
+    dest: int, source: int, chan_ident: int, io_port: int
+) -> bytes:
+    return _pack(0x0270, dest, source, param1=chan_ident, param2=io_port)
+
+
+def mot_set_postrigenstate(
+    dest: int, source: int, chan_ident: int, state: int
+) -> bytes:
+    return _pack(0x0272, dest, source, param1=chan_ident, param2=state)
+
+
+def mot_req_postrigenstate(dest: int, source: int, chan_ident: int) -> bytes:
+    return _pack(0x0273, dest, source, param1=chan_ident)
+
+
+def mot_set_lcddisplayparams(
+    dest: int,
+    source: int,
+    js_sensitivity: int,
+    disp_brightness: int,
+    disp_time_out: int,
+    disp_dim_level: int,
+) -> bytes:
+    data = struct.pack(
+        "<HHHHH", js_sensitivity, disp_brightness, disp_time_out, disp_dim_level, 0
+    )
+    return _pack(0x0543, dest, source, data=data)
+
+
+def mot_req_lcddisplayparams(dest: int, source: int, chan_ident: int) -> bytes:
+    return _pack(0x0544, dest, source, param1=chan_ident)
+
+
+def mot_set_lcdmoveparams(
+    dest: int,
+    source: int,
+    chan_ident: int,
+    js_mode: int,
+    jog_step_size: int,
+    accn: int,
+    max_vel: int,
+    jog_stop_mode: int,
+    preset_pos: int,
+) -> bytes:
+    data = struct.pack(
+        "<HH3lHlH",
+        chan_ident,
+        js_mode,
+        jog_step_size,
+        accn,
+        max_vel,
+        jog_stop_mode,
+        preset_pos,
+        0,
+    )
+    return _pack(0x0546, dest, source, data=data)
+
+
+def mot_req_lcdmoveparams(dest: int, source: int, chan_ident: int) -> bytes:
+    return _pack(0x0547, dest, source, param1=chan_ident)
+
+
+def mot_set_movesyncharray(
+    dest: int,
+    source: int,
+    array_id: int,
+    channels: int,
+    num_points: int,
+    start_ix: int,
+    time_pos: list[int],
```

Comment:
If we include a `list[int]` annotation, we probably want the `from __future__ import annotations` import to make sure we are compatible with some older python versions...

that or type hint as `Sequence[int]` to be a bit more permissive for static type checkers

Reply:
Have added the suggested import to aid compatibility